src/main.ts

```diff
@@ -6,6 +6,7 @@ import { AppModule } from './app.module';
 
 const port = process.env.PORT || 4000;
 
+
```

Comment:
add `coverage` folder to the .dockerignore - it generated by the `test:cov` script)